File: LeetCode/Simulation/JS/Leetcode1598-crawler-log-folder.js
var minOperations = function(logs) {
    let [stp,n] = [0,logs.length];
    for(const s of logs) {
        if(s == "./" || (s == "../" && stp == 0)) continue;
        if(s == "../") stp--;
        else stp++;
    }
    return stp;
};

File: LeetCode/Simulation/Rust/Leetcode1598-crawler-log-folder.rs
impl Solution {
  pub fn min_operations(logs: Vec<String>) -> i32 {
    let (n, mut stp) = (logs.len(), 0);
    for i in 0..n {
      if logs[i] == "./" || logs[i] == "../" && stp == 0 {
        continue;
      }
      if logs[i] == "../" {
        stp -= 1;
      } else {
        stp += 1;
      }
    }
    stp
  }
}
